nextcloud-startup-script.sh

```diff
@@ -493,11 +494,16 @@ do
             clear
             run_app_script talk
         ;;
-
+	
+        CookieLifetime)
+            clear
+            run_app_script cookielifetime
```

Comment:
Put it in static instead. This is no app. :)

Reply:
is moved over.